src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -27,15 +27,15 @@ redirects:
 
 import alertsAlertsandAiOverviewPage from 'images/alerts_screenshot-full_alerts-and-ai-overview-page.png'
 
-As the digital world continues to grow more tangled and complex, it's easy for DevOps teams to feel overwhelmed. There are so many new data points, dashboards, entities, and alerts that it can be hard for on-call teams to know what's happening where, and what to do about it. New Relics alerts and applied intelligence were designed to end response fatigue so your team can not only detect and diagnose incidents, but prioritize them for immediate action.
+As the digital world continues to grow more tangled and complex, it's easy for DevOps teams to feel overwhelmed. There are so many new data points, dashboards, entities, and alerts that it can be hard for on-call teams to know what's happening where, and what to do about it. New Relic's alerts and applied intelligence were designed to end response fatigue so your team can not only detect and diagnose incidents, but prioritize them for immediate action.
 
-Alerts allow you to define the conditions in your system you want to be notified about, letting you focus on the things you care about most. Applied intelligence is a machine-learning engine that alerts your team to potential problems by first identifying anomalous behavior, correlating any relevant incidents, alerting your team of active issues and providing a root cause analysis.
+Alerts allow you to define the conditions in your system you want to be notified about, letting you focus on the things you care about most. Applied intelligence is a machine-learning engine that notifies your team about potential problems by first identifying anomalous behavior, correlating any relevant incidents, alerting your team of active issues and providing a root cause analysis.
 
-Alerts and applied intelligence were created for busy DevOps teams: it reduces noise and makes sure you never miss an issue. 
+Alerts and applied intelligence were created for busy DevOps teams: they reduce noise and make sure you never miss an issue. 
 
 ## How it all works 
 
-Alerts and applied intelligence are a suite of tools and features that proactively notify you when things go wrong... right when they're starting to go wrong.
+Alerts and applied intelligence are a suite of tools and features that proactively notify you when things go wrong right when they're starting to go wrong.
```

Comment:
I think removing the ellipsis was the right call here but the impact of the repetition is lost without it to me personally. Maybe "right when they're starting to go wrong" could be "right when the problem begins" or something similar. Again, super granular, maybe not worth changing.

Reply:
I think that's a good callout, thanks!